src/App.vue

```diff
@@ -120,9 +126,25 @@ export default {
         account: address,
         balance: Util.atomsToAe(balance).toFixed(2),
       });
+
+      // trigger run async in background
+      this.loadTokenBalances(address);
```

Comment:
> called before wallet is initialized if the address was stored before

that's the goal of it, maybe I don't understand what you suggest. My idea was to fetch the balances without waiting for wallet, if wallet is found this will be done again with an eventually updated address